cmake/src.cmake

```diff
@@ -82,10 +97,10 @@ set(MATIO_PUBLIC_HEADERS
 if(STDINT_MSVC)
     set(MATIO_PUBLIC_HEADERS ${MATIO_PUBLIC_HEADERS} ${PROJECT_SOURCE_DIR}/visual_studio/stdint_msvc.h)
 endif()
-set_target_properties(matio PROPERTIES PUBLIC_HEADER "${MATIO_PUBLIC_HEADERS}")
+set_target_properties(${PROJECT_NAME} PROPERTIES PUBLIC_HEADER "${MATIO_PUBLIC_HEADERS}")
 
 # 'make install' to the correct locations (provided by GNUInstallDirs).
-install(TARGETS matio EXPORT libmatio
+install(TARGETS ${PROJECT_NAME} EXPORT libmatio
```

Comment:
You could change this to `lib${PROJECT_NAME}` as well, just like line 64.

Reply:
Done. Thanks.